.evergreen/atlas_data_lake/pull-mongohouse-image.sh

```diff
@@ -13,6 +13,11 @@ if [ ! -f secrets-export.sh ]; then
 fi
 source secrets-export.sh
 unset AWS_SESSION_TOKEN
-aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin $REPO
-docker pull $REPO
+if command -v docker &> /dev/null; then
```

Comment:
`run-server.sh` and `teardown.sh` check for `podman` before `docker`. Should this be consistent with that ordering?

Reply:
Updated